Libraries/LibGUI/FilePicker.h

```diff
@@ -31,10 +31,11 @@
 #include <LibCore/StandardPaths.h>
 #include <LibGUI/Dialog.h>
 #include <LibGUI/Image.h>
+#include <LibGUI/Model.h>
 
 namespace GUI {
 
-class FilePicker final : public Dialog {
+class FilePicker final : public Dialog, private ModelClient {
```

Comment:
```suggestion
class FilePicker final
    : public Dialog
    , private ModelClient {
```
I think this is usually formatted by `clang-format`, though I'm not sure. It's the Serenity style tho ^^